marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/core.sjs

```diff
@@ -225,6 +258,16 @@ function deleteArtifactSettings(artifactType, artifactName, artifactVersion = 'l
     return { success: true };
 }
 
+function getEntityNames() {
+    let entities = cts.search(cts.collectionQuery("http://marklogic.com/entity-services/models"))
```

Comment:
You can use "map" here to cut out a good chunk of code:

    cts.search(...).toArray().map(e => e.xpath("//info//title");